packages/sitecore-jss-nextjs/src/middleware/personalize-middleware.ts

```diff
@@ -127,7 +131,7 @@ export class PersonalizeMiddleware extends MiddlewareBase {
       cookieDomain: hostname,
       enableServerCookie: true,
     })
-      .addPersonalize()
+      .addPersonalize({ enablePersonalizeCookie: this.config.enablePersonalizeCookie ?? false})
```

Comment:
shouldn't we set it to true if it's undefined so that we will not have breaking changes? they can just set the value to false if they want to opt out.

Reply:
makes sense, I will update